### ISSUE TITLE
Menu bar text not visible

### ISSUE THREAD

**Issue by devansh08** [electron/electron#15194]
* Output of `node_modules/.bin/electron --version`: v2.0.11
* Operating System (Platform and Version): Arch Linux

**Expected Behavior**
Menu bar text font color should be white for a black/dark background.

**Actual behavior**
Menu bar text color is gray(close to black) with black background. So the text is unreadable.

**To Reproduce**
Have had this problem since installation.

**Screenshots**
![atom](https://user-images.githubusercontent.com/21225957/47019620-265fee00-d175-11e8-979b-355265503a47.png)
![code](https://user-images.githubusercontent.com/21225957/47019622-265fee00-d175-11e8-947b-1d2260c7e04d.png)

**Additional Information**
I am facing this problem on both Atom and VS Code so I am presuming it to be related to Electron. This is happening on both light and dark themes. In fact there is no change in the menu bar colors for Adapta and Adapta-Nokto themes. I have read abt this issue on another thread here but it was last updated an year ago and mentions the problem being related to electron not having GTK3 support. I presume this has been achieved so I dont know what to do now.

**Comment by ckerr** [comment]
Possibly related to #14813?

**Comment by devansh08** [comment]
#14813 seems to have been fixed in v2.0.11 according to release logs. But its not working for me on v2.0.11 or v2.0.12.

**Comment by devansh08** [comment]
This issue seems to only occur for VS Code and Atom installed from Pacman repos. Everything works fine for their respective AUR repos which seem to be created from the official binaries.

**Comment by sbatten** [comment]
Adding in this information from my duplicate for awareness.
> After tweaking the theme css for the Yaru Ubuntu 18.10 theme, it seems the foreground color for the menubar is being pulled from the context menu color, not the menubar foreground color.
The colors are set here: https://github.com/electron/electron/blob/74644e53e85afe05d99592f4633c40fa070b345a/atom/browser/ui/views/menu_bar.cc#L142

**Comment by devansh08** [comment]
I think this issue has been fixed but not added to any stable release yet. We may have to wait on that.

**Comment by codebytere** [comment]
@malnvenshorn no, it looks like it slipped under our radar: apologies! @Cobinja helpfully opened up backports, so we can look to get those in as soon as possible 🙇‍♀️

**Comment by DProBoy** [comment]
Is it fixed yet or something ? i still have this issue ... any work-arounds ?

**Comment by Cobinja** [comment]
The backports are merged into the 3-0-x and 4-0-x branches, but there were no point releases yet, so I assume it to arrive with the next ones for 3.0 and 4.0.
@codebytere what about the 3-1-x backport?

**Comment by Cobinja** [comment]
I did the backports only for the above mentioned branches because the original PR was tagged for them.
@codebytere: When will/did 2.0 reach end-of-life? Would it still make sense to backport it now?

**Comment by Synaxis** [comment]
Still happening ....

**Comment by jaynarayan89** [comment]
I also have this issue on fresh install of atom . I am using atom version 1.34.0 on ubuntu 18.10 x64.

**Comment by klvenky** [comment]
I am seeing this issue on ubuntu 19.04 on latest version (v1.38.x I guess). Is there some configuration that I can do to make this work. Just as someone already asked for, some setting option like in vscode

**Comment by devansh08** [comment]
In VS Code you can set Title Bar Style as custom rather than native. This will merge the two and make the menus visible. Although this setting seems to be default in the latest versions by my knowledge.

**Comment by klvenky** [comment]
Hi @devansh08,
I agree that it works that way in vscode however, I would just like to know if there is a workaround for atom. Would be great if there is a config setting that could help me in that :)

**Comment by klvenky** [comment]
this is working on other themes like advaita. However, for yaru theme the fix is in v1.39-beta2. I have checked and found that it works. :)

**Comment by timburgess** [comment]
The default in VS Code is for menus to be 'native'. Changing Settings for **Window: Title Bar Style** to be 'custom' is a workaround for this issue. I wasn't able to find a solution in Atom.
